build/lib/builder.js

```diff
@@ -38,9 +37,7 @@ function determineBabelOptions(babelOptions) {
 	const options = {
 		...babelOptions,
```

Comment:
<!--
  0 failure: 
  1 warning:  build/lib/builder...
  
  
  DangerID: danger-id-Danger;
  File: build/lib/builder.js;
  Line: 38;
-->

- :warning: build/lib/builder.js line 38 – Rest/spread properties are not supported until Node.js 8.3.0. The configured version range is '>=8'. (node/no-unsupported-features/es-syntax)